Sources/CustomAnnotatedMap/Models/MapRect.swift

```diff
@@ -59,3 +59,21 @@ extension MapRect: Hashable {
         hasher.combine(self.size.height)
     }
 }
+
+extension MapRect {
+    /// Compares two instances of MapRect with a certain precision
+    /// Precision is set due to slight inaccuracies in MKMap region property during map movements
+    /// - Parameter mapRect: MapRect to compare
+    /// - Parameter precision: Comparison precision - 0.0001 degrees ~= 10 meters
+    /// - Returns: The differences in centers and spans of the mapRects are smaller than 10 metres
+    func isSame(as mapRect: MapRect?, precision: CLLocationDegrees = 0.0001) -> Bool {
```

Comment:
I like the solution, but any reason to not address this logic on an `Equatable` conformance extension?

Reply:
I was thinking about it. This type already conforms to `Equatable` since it conforms to `Hashable`. I left the implementation of that as it is in case somebody writes `mapRect1 == mapRect2` and expects the comparison to be precise. 

With this one, I wanted to be explicit with the precision since this is very specific to comparing the rects while the mapView is displayed.